plugins/python-build/scripts/add_miniconda.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env/python3.7
```

Comment:
It might be little funny if a Python script packaged with pyenv is configured to skip using pyenv 😛

Reply:
Oops, that shouldn't have the slash before python.

I don't think this necessarily skips using pyenv though - wouldn't `/usr/bin/env python3.7` use whichever python3.7 is first in the path, including whatever pyenv shims in?

Reply:
It actually depends on how `$PATH` is configured for the caller's shell, however, having `/usr/bin/env python3.7` in shebang would work sufficiently for the purpose.